src/app/components/language-translation/language-translation.module.ts

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
`components` folder is intended for the components.
modules are on the top level folder (they do not have a proper subfolder), so this file should be just moved 1 level up